arangod/RestServer/EnvironmentFeature.cpp

```diff
@@ -44,8 +44,30 @@
 
 #ifdef __linux__
 #include <sys/sysinfo.h>
+#include <unistd.h>
 #endif
 
+namespace {
+std::string_view trimProcName(std::string_view const& content) {
```

Comment:
string_views are very cheap to pass by value, so there is no need to pass them by reference.
```suggestion
std::string_view trimProcName(std::string_view content) {
```